style-guide/src/components/section-header/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import styles from './styles.module.css';
+
+import token from 'otkit-desktop-typography/token.common';
+
+export default function SectionHeader({ text }) {
+	return <h2 className={styles['section-header']}>{text}</h2>;
```

Comment:
It's unfortunate that CSS Modules in Gatsby does not enable exporting class names in camel case.

Reply:
I think that you should be able to extend the configuration of the gatsby webpack to add support for that if you feel the need - https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/add-custom-webpack-config.md

Reply:
@nickbalestra , is this PR not rebased against your prettier PR we just merged? Doesn't prettier do 2 spaces by default?